code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -117,37 +117,25 @@
 	robotize()
 	..()
 
-/obj/item/organ/internal/optical_sensor
+/obj/item/organ/internal/eyes/optical_sensor
```

Comment:
[No, it's linked to the vision organ not being a subtype of eyes.](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/carbon/carbon.dm#L311-L313)